examples/example_lib_window.ks

```diff
@@ -0,0 +1,43 @@
+run lib_window_number_dialog.
+run lib_window_vessel_stats.
+run lib_window_menu.
+run lib_process.
+
+clearscreen.
+ag1 off.
+
+set fraction to 0.5.//fraction of screen to left window
+set first_window_share to round((terminal:width-3)*fraction).
+set window1 to create_window(0,0,
+	first_window_share,terminal:height-3).
+set window2 to create_window(window1[0]+window1[2]+1,0,
+	terminal:width-window1[0]-window1[2]-3,terminal:height-3).
+// if you have just one window, just use:
+// create_window(0,0,terminal:width-2,terminal:height-2)
+// instead of those six lines
+
+set all_proc to list().
+//all_proc:add(open_window_menu("Select stuff:",
+//	list("Argh","Nope","yelp"),window1)).
+all_proc:add(open_window_number_dialog("Apoapsis: ",100000,window1)).
+all_proc:add(open_window_vessel_stats(window2)).
+
+set old_terminal_width to terminal:width.
+set old_terminal_height to terminal:height.
+until false{
```

Comment:
For now, yes. But you can add a check if process list is empty. Also, remember that this example is just a toy model. There are a few bugs because of laziness, such as crashing if one tries to resize terminal after closing one of them - there is hardcoded number 2, for two windows.